_includes/people.html

```diff
@@ -1,6 +1,6 @@
 <div class="card people-card" id="{{ username }}">
   <div class="card-content">
-    <div class="media">
+    <div class="media" style="height: 230px;">
```

Comment:
We would put the height in the CSS maybe with a dedicated class

Reply:
Done. 👍